flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/table/KinesisOptions.java

```diff
@@ -44,26 +49,54 @@
 		private KinesisOptions() {
 		}
 
-		// --------------------------------------------------------------------------------------------
+		// -----------------------------------------------------------------------------------------
 		// Kinesis specific options
-		// --------------------------------------------------------------------------------------------
+		// -----------------------------------------------------------------------------------------
 
 		/**
-		 * Prefix for properties delegated to the
+		 * Prefix for properties defined in
+		 * {@link org.apache.flink.streaming.connectors.kinesis.config.AWSConfigConstants}
+		 * that are delegated to
 		 * {@link org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer} and
 		 * {@link org.apache.flink.streaming.connectors.kinesis.FlinkKinesisProducer}.
 		 */
-		public static final String PROPERTIES_PREFIX = "properties.";
+		public static final String AWS_PROPERTIES_PREFIX = "aws.";
+
+		/**
+		 * Prefix for properties defined in
+		 * {@link org.apache.flink.streaming.connectors.kinesis.config.ConsumerConfigConstants}
+		 * that are delegated to
+		 * {@link org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer}.
+		 */
+		public static final String CONSUMER_PREFIX = "source.";
```

Comment:
I'm very sorry. Somehow I missed this in the Google docs document. Instead of `source` we should use `scan` according to FLIP-122. Because we have two types of sources, we should make it clear that all these properties related to a table scan and not a lookup source.

Reply:
No problem.